Html/users.html

```diff
@@ -1,174 +1,174 @@
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-			<div class="col-sm-2">
-				<div class="panel panel-default userBox">
-					<div class="panel-body">
-						<img src="../Images/useravatar.png">
-						<button type="button" class="btn btn-primary">Follow</button>
-						<p>Marty lachtong</p>
-					</div>
-				</div>
-			</div>
-		</div>
-	</div>
-
-
-<div class="row row-m-b">
-	<div class="col-sm-2 Followees">
-		<div class="thumbnail">
-				<div class="panel-body">
-					<b>Followees</b>
-				</div>
-			<div class="panel panel-default FolloweesBox">
-				<div class="panel-body">
-					<img src="../Images/useravatar.png"><br>
-					<button type="button" class="btn btn-primary">Unfollow</button><br>
-					<p>Marty lachtong</p>
-				</div>
-			</div>
-			<div class="panel panel-default FolloweesBox">
-				<div class="panel-body">
-					<img src="../Images/useravatar.png"><br>
-					<button type="button" class="btn btn-primary">Unfollow</button><br>
-					<p>Marty lachtong</p>
-				</div>
-			</div>
-		</div>
-	</div>
-</div>
-
-
-
-
-
-
-
-</body>
+<!DOCTYPE html>
+<html>
+<head>
+	<title>OfekTwitter</title>
+	<link rel = "stylesheet"
```

Comment:
no need to put every property in a new line - should be in one line, or it can get too long